File: python/caesar cipher/encrypt.py
def encrypt_caesar(plaintext):
    for key in range(1, 27):
        ciphertext = ''
        for c in plaintext:
            if c.isalpha():
                base = ord('A') if c.isupper() else ord('a')
                encrypted = chr(base + (ord(c) - base + key) % 26)
                ciphertext += encrypted
            else:
                ciphertext += c  # Non-alphabetic characters remain unchanged
        print(f"Key {key} output: {ciphertext}")

def main():
    plaintext = input("Enter the plaintext: ")
    encrypt_caesar(plaintext)

if __name__ == "__main__":
    main()

File: c++/caesar cipher/encrypt.cpp
#include <iostream>
#include <string>

void encryptCaesar(const std::string& plaintext, int shift) {
    for (int key = 1; key <= 26; ++key) {
        std::string ciphertext;
        for (char c : plaintext) {
            if (std::isalpha(c)) {
                char base = std::isupper(c) ? 'A' : 'a';
                char encrypted = base + (c - base + key) % 26;
                ciphertext += encrypted;
            } else {
                ciphertext += c; // Non-alphabetic characters remain unchanged
            }
        }
        std::cout << "Key " << key << " output: " << ciphertext << '\n';
    }
}

int main() {
    std::string plaintext;
    std::cout << "Enter the plaintext: ";
    std::cin >> plaintext;

    encryptCaesar(plaintext, 0); // Brute-force all keys
    return 0;
}
